pybamm/parameters/parameter_values.py

```diff
@@ -65,7 +65,10 @@ def __init__(self, values, chemistry=None):
                 values.pop("chemistry", None)
                 self.update(values, check_already_exists=False)
             else:
-                raise ValueError("Invalid Parameter Value")
+                valid_sets = "\n".join(pybamm.parameter_sets.keys())
+                raise ValueError(
+                    f"{values} is not a valid parameter set. Parameter set must be one of:\n {valid_sets} "
```

Comment:
```suggestion
                    f"{values} is not a valid parameter set. Parameter set must be one of:\n{valid_sets} "
```

Reply:
There should also be single quotes or something around `{values}` to make the error message nicer:

For example the test is: "Junk is not a valid parameter set."
It would probably be nicer to have "'Junk' is not a valid parameter set."

Reply:
Just an update to Rob's suggestion. I removed the trailing space and added the quotes. I also updated the test to match
```suggestion
                    f"'{values}' is not a valid parameter set. Parameter set must be one of:\n{valid_sets}"
```